### PR TITLE
Staff info and cert

### PR DESCRIPTION
**Pull request by kevinverhoff**
A lot of big updates here:

1. Unpivoting the staff info cert feed
2. Making a long feed of certifications
3. Making a feed of staff info links
4. Updating staff_roster_history so it pulls data from the staff info feed
5. Updating tableau.staff_roster to include new fields
6. Updating attrition feed to include new fields
7. BONUS: Charlie set up a feed for files to flow into google forms submissions!

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205223770984570

### CODE REVIEW

File: src/dbt/kipptaf/models/google/forms/base/base_google_forms__staff_info_responses_wide.sql
Comment (cbini) [inline src/dbt/kipptaf/models/google/forms/base/base_google_forms__staff_info_responses_wide.sql:1]: move to people

File: src/dbt/kipptaf/models/google/forms/base/base_google_forms__staff_info_responses_wide.sql [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+            nj_cert_issue_date_1,
+            fl_cert_endorsement_3,
+            cert_steps_taken,
+            nj_cert_endorsement_4,
+            cert_out_of_state_details,
+            gender_identity,
+            level_of_education,
+            years_exp_outside_kipp
+
+        from `teamster-332318.kipptaf_surveys.src_surveys__staff_info_certification`
```
Comment (cbini) [inline src/dbt/kipptaf/models/google/forms/base/base_google_forms__staff_info_responses_wide.sql:265]: use dbt ref

File: src/dbt/kipptaf/models/google/forms/base/base_google_forms__staff_info_responses_wide.sql [hunk 2]
```diff
@@ -0,0 +1,273 @@
+with
+    simple_view as (
+        select
+            cast(
+                replace(replace(last_submitted_time, 'T', ' '), 'Z', '') as datetime
```
Comment (cbini) [inline src/dbt/kipptaf/models/google/forms/base/base_google_forms__staff_info_responses_wide.sql:5]: fix on form_responses

File: src/dbt/kipptaf/models/people/base/base_people__staff_roster_history.sql [hunk 3]
```diff
@@ -375,12 +375,67 @@ with
         left join
             {{ ref("stg_ldap__user_person") }} as ldap
             on en.employee_number = ldap.employee_number
+    ),
+    with_info_survey as (
```
Comment (cbini) [inline src/dbt/kipptaf/models/people/base/base_people__staff_roster_history.sql:379]: can this be added into an earler cte?

File: src/dbt/kipptaf/models/people/staging/stg_people__certifications_survey_long.sql
Comment (cbini) [inline src/dbt/kipptaf/models/people/staging/stg_people__certifications_survey_long.sql:1]: rename (remove survey long)

File: src/dbt/kipptaf/models/people/staging/stg_people__certifications_survey_long.sql [hunk 4]
```diff
@@ -0,0 +1,173 @@
+with certifications_long AS (
```
Comment (cbini) [inline src/dbt/kipptaf/models/people/staging/stg_people__certifications_survey_long.sql:1]: i think we can do this with an unpivot

File: src/dbt/kipptaf/models/people/staging/stg_people__rn_staff_info_feed_and_links.sql
Comment (cbini) [inline src/dbt/kipptaf/models/people/staging/stg_people__rn_staff_info_feed_and_links.sql:1]: is this a tableau view?

**Review by cbini — CHANGES_REQUESTED** [review]
gonna make these changes myself, but need your input on the last view. is that meant for tableau?

File: src/dbt/kipptaf/models/google/forms/base/base_google_forms__staff_info_responses_wide.sql
Comment (cbini) [reply src/dbt/kipptaf/models/google/forms/base/base_google_forms__staff_info_responses_wide.sql:1]: moved to surveys instead

File: src/dbt/kipptaf/models/google/forms/base/base_google_forms__staff_info_responses_wide.sql (hunk 2 above)
Comment (cbini) [reply src/dbt/kipptaf/models/google/forms/base/base_google_forms__staff_info_responses_wide.sql]: unnecessary, need to parse archive timestamp correctly

File: src/dbt/kipptaf/models/surveys/intermediate/int_surveys__staff_information_survey_pivot.sql [hunk 5]
```diff
@@ -0,0 +1,292 @@
...
+    mo.relay_status,
+from response_pivot as rp
+inner join multiselect_options as mo on rp.response_id = mo.response_id
+
+-- union all
+
+-- select
+--     null as response_id,
+--     respondent_email,
+--     cast(timestamp as datetime format 'MM/DD/YYYY HH24:MI:SS') as last_submitted_time,
```
Comment (cbini) [inline src/dbt/kipptaf/models/surveys/intermediate/int_surveys__staff_information_survey_pivot.sql:233]: figure out what's going on here

File: src/dbt/kipptaf/models/surveys/intermediate/int_surveys__staff_information_survey_pivot.sql [hunk 6]
```diff
@@ -0,0 +1,292 @@
...
+
+--     {# multiselect_options #}
+--     alumni_status,
+--     community_grew_up,
+--     community_professional_experience as community_professional_exp,
+--     languages_spoken,
+--     path_to_education,
+--     race_ethnicity,
+--     relay_status,
+-- from `teamster-332318.kipptaf_surveys.src_surveys__staff_info_certification`
```
Comment (cbini) [inline src/dbt/kipptaf/models/surveys/intermediate/int_surveys__staff_information_survey_pivot.sql:292]: uncomment when ts fixed

**Review by cbini — CHANGES_REQUESTED** [review]

File: src/dbt/kipptaf/models/people/staging/stg_people__certifications_survey_long.sql (hunk 4 above)
Comment (cbini) [reply src/dbt/kipptaf/models/people/staging/stg_people__certifications_survey_long.sql]: not a priority, added TODO

File: src/dbt/kipptaf/models/people/staging/stg_people__rn_staff_info_feed_and_links.sql
Comment (kevinverhoff) [reply src/dbt/kipptaf/models/people/staging/stg_people__rn_staff_info_feed_and_links.sql:1]: yes, and for an appsheet tool.

**Review by kevinverhoff — COMMENTED** [review]
I think I caught everything here. Seems like I confused some of the base vs. int vs etc. stuff? Did I miss anything?

**Review by cbini — APPROVED** [review]

**Pull request merged**